pwndbg/gdblib/stack.py

```diff
@@ -135,3 +136,28 @@ def is_executable() -> bool:
             nx = True
 
     return not nx
+
+
+def yield_return_addresses():
+    sp = pwndbg.gdblib.regs.sp
+    stack = pwndbg.gdblib.vmmap.find(sp)
+
+    # Enumerate all return addresses
+    frame = gdb.newest_frame()
```

Comment:
What happens if GDB can't figure out stack frames?

Reply:
not sure :man_shrugging: this pattern is used in a lot of places ( just getting the frame and assuming its valid ) fwiw